### PR TITLE
ROX-19605: new feature-flag to control aggregation of unmatched IPs

### PR DESCRIPTION
**Pull request by ovalenti**
## Description

A new feature-flag ROX_ENABLE_EXTERNAL_IPS environment variable controls whether external network connections which do not match any defined known network are aggregated in the default bucket.

The default value is false, which corresponds to the previous behavior. 

## Checklist
- [x] Investigated and inspected CI test results
- [ ] Updated documentation accordingly

**Automated testing**
  - [x] Added unit tests

### CODE REVIEW

**Review by Molter73 — APPROVED** [review]
LGTM!

**Comment by ovalenti** [comment]
Since the other parts of the system are not going to play nice with only this change in collector, it looks like we will need a feature-flag that covers several places. I renamed it and also reverted the logic.

Could you please take an other look and see if everything's fine, @Molter73 ?

File: collector/lib/ConnTracker.cpp [hunk 1]
```diff
@@ -91,7 +91,11 @@ IPNet ConnectionTracker::NormalizeAddressNoLock(const Address& address) const {
     return network;
   }
 
-  // Otherwise, associate it to "rest of the internet".
+  if (enable_external_ips_) {
+    return IPNet(address, 0, true);
+  }
+
+  // associate it to "rest of the internet".
```
Comment (Molter73) [inline collector/lib/ConnTracker.cpp:98]: [ultimate-nit] We can now leave the comment as it was.

**Review by Molter73 — APPROVED** [review]
I actually like this approach better, LGTM!

**Comment by Molter73** [comment]
On a separate note, mind updating the PR description? It still names the old env variable.

**Pull request merged**